Sources/CallWithDoxDialer/CallWithDoxDialer.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+#if canImport(UIKit)
```

Comment:
This wasn't required for integration into another project, but I couldn't get `swift build` to pass validation unless I added this check.

Reply:
This feels to me like it's trying to remain valid on other platforms somewhere (MacOS)? but not worth a ton of time to figure out if it's working fine. I saw that it has `iOS` as the platform in the package config. 🤷🏼‍♂️